ckanext/saml2/saml2_mapper.py

```diff
@@ -0,0 +1,25 @@
+role_equality = dict(
+    member='member',
+    reader='member',
```

Comment:
`reader` role is just a quirk of the proof of concept.  The shared spec is to use only `member|editor|admin'